.pylintrc

```diff
@@ -319,7 +319,7 @@ ignore-imports=no
 ignore-signatures=no
 
 # Minimum lines number of a similarity.
-min-similarity-lines=4
+min-similarity-lines=20
```

Comment:
Why it has been triggered?

Reply:
By explicit command imports. This rule gives too many false flags, and it can't be ignored from the file level.